code-challenges.js

```diff
@@ -40,12 +63,36 @@ var testString1 = "albatross"
 var testString2 = "jabberwocky"
 // Expected output: "jAbBeRwOcKy"
 
+const capitalized = (string) => {
+  var newString = ""
+  for(let i = 0; i < string.length; i++) {
+    if(i % 2 == 0){
+      newString += string.charAt(i).toUpperCase()
+    }else {
+      newString += string.charAt(i)
+    }
+  }
+  return newString
+}
 
-
+console.log(capitalized(testString1))
+console.log(capitalized(testString2))
 
```

Comment:
Again, you are super close. It looks like your logic on which index to capitalize is just backwards.